common/static/common/js/components/views/paging_header.js

```diff
@@ -1,113 +1,64 @@
-define(["underscore", "backbone", "gettext", "text!common/templates/components/paging-header.underscore"],
```

Comment:
I assume this comment has to do with preserving history as well, so I'll address it when you get into the office.